msal/application.py

```diff
@@ -1958,9 +1964,11 @@ def __init__(
                  if your app is expected to run on Windows 10+
                * ``msauth.com.msauth.unsignedapp://auth``
                  if your app is expected to run on Mac
+               * ``ms-appx-web://Microsoft.AAD.BrokerPlugin/your_client_id``
+                 if your app is expected to run on Linux, especially WSL
 
             2. installed broker dependency,
-               e.g. ``pip install msal[broker]>=1.31,<2``.
+               e.g. ``pip install msal[broker]>=1.32,<2``.
```

Comment:
Since we shipped MSAL 1.32 just yesterday, all the mentioning of "1.32" shall be changed to "1.33" in this PR. Sorry for the inconvenience. :-)